packages/plugins/src/plugins/eigenlayer/fetchers/deposit.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  ethereumNativeAddress,
+  ethereumNetwork,
+  NetworkId,
+  PortfolioAsset,
+  PortfolioElement,
+  PortfolioElementType,
+} from '@sonarwatch/portfolio-core';
+import { getEvmClient } from '../../../utils/clients';
+import { Cache } from '../../../Cache';
+
+import { eigenPodManagerAddress, platformId } from '../constants';
+import { getAddress } from 'viem';
+import { abi } from '../abi';
+import tokenPriceToAssetToken from '../../../utils/misc/tokenPriceToAssetToken';
+import BigNumber from 'bignumber.js';
+
+/**
+ * Returns the deposit positions for a given owner
+ * @param owner - The address of the owner
+ * @param cache - The cache instance
+ * @returns The deposit positions
+ */
+export const getDepositPositions = async (owner: string, cache: Cache) => {
+  const client = getEvmClient(NetworkId.ethereum);
+  const podManagerAddress = getAddress(eigenPodManagerAddress);
+
+  const podUserAddress = await client.readContract({
+    address: podManagerAddress,
+    abi: [abi.ownerToPod],
+    functionName: abi.ownerToPod.name,
+    args: [getAddress(owner)],
+  });
+
+  // If the user has no pod, the contract returns the ethereum native address address
+  if (!podUserAddress || podUserAddress === ethereumNativeAddress) return [];
+
+  // Get the balance of the pod user contract
+  const balance = await client.getBalance({
+    address: podUserAddress,
+  });
+
+  const assets: PortfolioAsset[] = [];
+  const tokenPrice = await cache.getTokenPrice(
+    ethereumNativeAddress,
+    NetworkId.ethereum
```

Comment:
Let's use a single local variable in the method to avoid directly referring to the constant so many times. This will make our lives much more easier when we get to support more chains.

```suggestion
    network
```